change-notes/1.21/analysis-python.md

```diff
@@ -3,8 +3,33 @@
 
 ## General improvements
 
-> Changes that affect alerts in many files or from many queries
-> For example, changes to file classification
+Points-to analysis has been re-implemented to support more language features and provide better reachability analysis.
+The new implementation adds the following new features:
+
+* Non-local tracking of bound methods and instances of `super()`
+* Superior analysis of conditionals and thus improved reachability analysis.
+* Superior modelling of descriptors, for example, classmethods and staticmethods.
```

Comment:
modelling -> modeling

Are "classmethods" and "staticmethods" code elements, or should they each be two words?

Reply:
They are code elements